components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue

```diff
@@ -79,7 +82,8 @@ const cssActive = (value) => {
   }
 
   &-filter {
-    column-gap: 2rem;
+    column-gap: 1.5rem;
```

Comment:
I think the columm-gap is too large compare with the design
![image](https://user-images.githubusercontent.com/22791238/224277437-1e651e8f-5541-4a76-8982-844b4aa3be37.png)